<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="mainTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>711</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>World</string>
    </attribute>
    <widget class="QPushButton" name="buttonTest">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonUpdateLists">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Lists</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listLocations">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonLocationInfo">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Location Info</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listActions">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonItemInfo">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Item Info</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listPeople">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QTabWidget" name="infoTabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>631</width>
       <height>421</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPerson">
      <attribute name="title">
       <string>Person</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Items</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listPersonItems">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAddItem">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRemoveItem">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Item</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCurrentPerson">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Person</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>46</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Health</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="personMoneyInput">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="personHealthInput">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
      <widget class="QListWidget" name="listPersonRelationship">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Relationships</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="personRelStrengthInput">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="personRelTypeInput">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
      <widget class="QLabel" name="personRelType">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="personRelStrength">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>120</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QLabel" name="personRelPrintout">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QListWidget" name="listPersonLocations">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>60</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>40</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Possible Locations</string>
       </property>
      </widget>
      <widget class="QLabel" name="personCurrentLocation">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Location:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="personMoodInput">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>10</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>46</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Mood</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAddRelationship">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Rel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonRemoveRelationship">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Rel</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>391</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>Cops and Robbers</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>141</width>
          <height>116</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Police Reputation</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="policeRep">
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Mob Reputation</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="mobRep">
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Police Wanted</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="policeWanted">
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Mob Wanted</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="mobWanted">
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Story Rank</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="storyRank">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="rankLabel">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Rank</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="citizenCheckBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>70</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Citizen</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabItem">
      <attribute name="title">
       <string>Item</string>
      </attribute>
      <widget class="QLabel" name="labelCurrentItem">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Item</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listItemActionDo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listItemActionGet">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>130</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Actions Do</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Actions Get</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonItemAddDo">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Do</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonItemRemoveDo">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Do</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonItemAddGet">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Get</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonItemRemoveGet">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove Get</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLocation">
      <attribute name="title">
       <string>Location</string>
      </attribute>
      <widget class="QListWidget" name="listLocationsGoTo">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelLocationCurrent">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Current Location:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Possible Locations</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listLocationsComeFrom">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Come From</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listLocationsPeople">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Current People Here</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabAction">
      <attribute name="title">
       <string>Action</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonPersonInfo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Person Info</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWorldObjects">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonActionInfo">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Action Info</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
